exercises/parallel-letter-frequency/sequential-letter-frequency/src/lib.rs

```diff
@@ -0,0 +1,49 @@
+use std::collections::HashMap;
+
+/// Simple sequential char frequency. Can it be beat?
+pub fn frequency(texts: &[&str]) -> HashMap<char, usize> {
+    let mut map = HashMap::new();
+
+    for line in texts {
+        for chr in line.chars().filter(|c| c.is_alphabetic()) {
+            if let Some(c) = chr.to_lowercase().next() {
+                (*map.entry(c).or_insert(0)) += 1;
+            }
+        }
+    }
+
+    map
+}
+
+// Poem by Friedrich Schiller. The corresponding music is the European Anthem.
```

Comment:
This might not be the ideal place to put test fixtures. 

I could them to an rs file outside `src`, `benches` and `tests` and pull it in with [`include!("path/to/a/file")`](https://doc.rust-lang.org/std/macro.include.html).

Or use some other fixture pattern I'm not aware of.